.travis.yml

```diff
@@ -15,12 +15,10 @@ install: |
     # Required for Racer autoconfiguration
     rustup component add rust-src
     rustup component add rust-analysis
-script: |
-  #!/bin/bash
-  set -e
```

Comment:
Why are these removed? Does travis early returns on error internally?

Reply:
Using `-` separated steps yes. When using a full script with `|` nope.

_"When one of the steps fails, the build stops immediately and is marked as errored."_

Reply:
Great, nice catch! :+1: